common/app/templates/inlineJS/nonBlocking/emailIframeTracking.scala.js

```diff
@@ -0,0 +1,30 @@
+@()
+
+const sendEvent = (payload, eventType) => {
+    const msg = {
+        id: 'xxxxxxxxxx'.replace(/x/g, () =>
+            // eslint-disable-next-line no-bitwise
+            ((Math.random() * 36) | 0).toString(36)
+        ),
+        type: `ophan-iframe-${eventType}`,
+        iframeId: window.frameElement ? window.frameElement.id : null,
+        value: payload,
+    };
+    window.parent.postMessage(msg, '*');
```

Comment:
Apologies for resurrecting this PR, but is there a reason that a `targetOrigin` can't be set? E.g.:

```js
window.parent.postMessage(msg, 'https://www.theguardian.com');
```

Reply:
At the time we definitely discussed it, but for the life of me I can't remember what we said now. It may potentially have been to do with using it with apps but we know that is no longer an issue. It's probably a good idea to change this over. I'll create a card and discuss it with the team to make sure I've not forgotten anything key.

Reply:
Could it be ye old Http to file system security context thingy? 
As in it's okay to put `*` because we expect to switch from `https://` to `file://`, and there's no other value in there that allows it?